SimTracker/SiPhase2Digitizer/plugins/Pixel3DDigitizerAlgorithm.h

```diff
@@ -0,0 +1,69 @@
+#ifndef _SimTracker_SiPhase2Digitizer_Pixel3DDigitizerAlgorithm_h
+#define _SimTracker_SiPhase2Digitizer_Pixel3DDigitizerAlgorithm_h
+
+//-------------------------------------------------------------
+// class Pixel3DDigitizerAlgorithm
+//
+// Specialization of the tracker digitizer for the 3D pixel
+// sensors placed at PXB-Layer1 (and possibly Layer2), and
+// at PXF-Disk1 and Disk2
+//
+// Authors: Jordi Duarte-Campderros (CERN/IFCA)
+//          Clara Lasaosa Garcia (IFCA)
+//--------------------------------------------------------------
+
+#include "SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.h"
+
+// Data formats
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+
+// system
+#include <functional>
+
+class Pixel3DDigitizerAlgorithm : public Phase2TrackerDigitizerAlgorithm {
```

Comment:
Should premixing work out of the box? Or is it something to be considered later?

Reply:
@makortel, the class is behaving exactly as the digitizer for planar (SimTracker/SiPhase2Digitizer/plugins/PixelDigitizerAlgorithm.h ), so If that class is working, this new class should as well.

Reply:
Ok, good, thanks.